lib/grape/validations/validators/length_validator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Grape
+  module Validations
+    module Validators
+      class LengthValidator < Base
+        def initialize(attrs, options, required, scope, **opts)
+          @min_length = options[:min]
```

Comment:
Call this `@min` and `@max` since we're already inside a `LengthValidator`

Reply:
done 👍